t/admin/schema-validate.t

```diff
@@ -398,3 +398,42 @@ location /t {
 --- error_code: 400
 --- response
 {"error_msg":"allOf 1 failed: value should match only one schema, but matches none"}
+
+
+
+=== TEST 13: Check node_schema optional port
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes',
+                ngx.HTTP_POST,
+                {
+                    uri = "/ip",
```

Comment:
you could search `127.0.0.1:1980` in the project, and find what uri is better

Reply:
OK, done. thanks